src/frontend-pwa/src/components/Header/Header.tsx

```diff
@@ -20,15 +18,17 @@ export default function Header() {
   return (
     <HeaderWrapper>
       <Banner>
-        <Link to="/">
-          <Image src={logo} alt="Go to the Home page" />
-        </Link>
+        <RoutingLink
+          content={<Image src={logo} alt="Go to the Home page" />}
+          path="/"
+        />
         <Heading>Wayfinder</Heading>
       </Banner>
       <div>
-        <Link to="/settings">
-          <SettingsNavButton />
-        </Link>
+        <RoutingLink
```

Comment:
Why not just use Link?

Reply:
@BradyMitch Sorry, there was a misunderstanding about what was meant. Need to rework this.